shared_model/bindings/model_crypto.cpp

```diff
@@ -21,8 +21,31 @@
 namespace shared_model {
   namespace bindings {
     crypto::Keypair ModelCrypto::generateKeypair() {
-      // TODO: 06.12.2017 luckychess create keypair generation interface IR-684
       return crypto::CryptoProviderEd25519Sha3::generateKeypair();
     }
+
+    crypto::Keypair ModelCrypto::generateKeypair(const std::string &seed) {
+      return crypto::CryptoProviderEd25519Sha3::generateKeypair(
```

Comment:
Add some using stuff here, it's cpp file

Reply:
I think we can create a template for generateKeypair() method with default parameter "Ed25519sha3".